### PR TITLE
Add support for capabilities on the `Account` resource

### PR DESCRIPTION
**Pull request by remi-stripe**
cc @stripe/api-libraries

### CODE REVIEW

File: account_test.go [hunk 1]
```diff
@@ -160,6 +164,10 @@ func TestAccount_Unmarshal(t *testing.T) {
 
 	assert.Equal(t, "acct_123", account.ID)
 
+	assert.Equal(t, stripe.AAccountCapabilityStatusActive, account.Capabilities.CardPayments)
+	// TODO: Figure out how to test that account.Capabilities.LegacyPayments is null/not present.
```
Comment (remi-stripe) [inline account_test.go]: @brandur-stripe Do you know how to approach this in Go?

Comment (brandur-stripe) [reply account_test.go]: Yeah, it's a little hacky, but basically `AccountCapability` is defined as a type based off string, which means that like string, it's possible for it to have an empty value `""` which is also the zero value for the string type. When `json` sees an unset/null value for a non-pointer type like string, it sets it to the type's zero value, so the equivalent of `""`.

Therefore, you can check an empty by converting an empty string to an `AccountCapabilityStatus` like this:

``` go
assert.Equal(t, AccountCapabilityStatus(""), account.Capabilities.LegacyPayments)
```

Comment (remi-stripe) [reply account_test.go]: gotcha. Is that clean/go-like though? Or would it be better to have a default constant `AccountCapabilityStatusNone` mapping to `""`?

**Pull request merged**